docs/src/reference/gremlin-variants.asciidoc

```diff
@@ -525,27 +525,43 @@ can be found in GitHub link:https://github.com/apache/tinkerpop/tree/x.y.z/greml
 and are designed to connect to a running <<gremlin-server,Gremlin Server>> configured with the
 `conf/gremlin-server.yaml` and `conf/gremlin-server-modern.yaml` files as included with the standard release packaging.
 
-To run the examples, first download an image of Gremlin Server from Docker Hub:
+This guide assumes Gremlin Server will be executed using Docker. Alternatively, Gremlin Server can run locally (see the <<gremlin-server,Gremlin Server>> documentation for this option).
+
+To start Gremlin Server using Docker, first download an image of Gremlin Server from Docker Hub:
+
 [source,shell]
 ----
 docker pull tinkerpop/gremlin-server
 ----
 
-The remote connection and basic Gremlin examples can be run on a clean server, which uses the default configuration file
-`conf/gremlin-server.yaml`. To start a clean server, launch a new container with `docker run`:
+Clean server:
+
 [source,shell]
 ----
 docker run -d -p 8182:8182 tinkerpop/gremlin-server
 ----
 
-The traversal examples should be run on a server configured to start with the Modern toy graph, using `conf/gremlin-server-modern.yaml`.
-To start a server with the Modern graph preloaded, launch a new container with `docker run`:
+Modern toy graph server:
+
 [source,shell]
 ----
 docker run -d -p 8182:8182 tinkerpop/gremlin-server conf/gremlin-server-modern.yaml
 ----
 
-Each example can now be run with the following commands:
+The remote connection and basic Gremlin examples can be run on a clean server, while traversal examples should be run on a server with the Modern graph preloaded.
```

Comment:
Nit: Try to maintain consistency in line length with the rest of the asciidoc files. This should break at 120 characters.